app/src/main/java/ca/marshallasch/veil/database/KnownPostsContract.java

```diff
@@ -13,10 +13,13 @@
 public final class KnownPostsContract {
 
     // DB query string
-    static final String SQL_CREATE_KNOWN_POSTS = "CREATE TABLE " +  KnownPostsEntry.TABLE_NAME + "(" +
-            KnownPostsEntry._ID + " INTERGER PRIMARY KEY," +
-            KnownPostsEntry.COLUMN_COMMENT_HASH + " VARCHAR(36) UNIQUE," +
-            KnownPostsEntry.COLUMN_POST_HASH + " VARCHAR(36))";
+    static final String SQL_CREATE_KNOWN_POSTS = "CREATE TABLE " +  KnownPostsEntry.TABLE_NAME +
+            "(" +
+            KnownPostsEntry._ID + " INTEGER PRIMARY KEY," +
+            KnownPostsEntry.COLUMN_COMMENT_HASH + " VARCHAR(36)," +
+            KnownPostsEntry.COLUMN_POST_HASH + " VARCHAR(36)," +
+            "UNIQUE(" + KnownPostsEntry.COLUMN_POST_HASH + ", " + KnownPostsEntry.COLUMN_COMMENT_HASH + ")" +
```

Comment:
Why is this line added and why was UNIQUE removed from COLUMN_COMMENT_HASH

Reply:
it need to be removed because it is not the comment hash that is unique it is the combination of post and comment hashes, also if you insert null's then the UNIQUE constraint gets ignored